pedalboard/version.txt

```diff
@@ -0,0 +1 @@
+0.3.6
```

Comment:
We might want this file to have a `py` extension.
For the content, we could insert something like:

```python
MAJOR = 0
MINOR = 3
PATCH = 6
__version__ = '%d.%d.%d' % (MAJOR, MINOR, PATCH)
```

Reply:
So, the `__version__` should be placed in the separated file?

Reply:
rename the file from `version.txt` to `version.py` and place the content I wrote above